### PR TITLE
Allow mobile workers to view Inventory by Location report

### PR DESCRIPTION
**Pull request by gcapalbo**
http://manage.dimagi.com/default.asp?235140
@dannyroberts

### CODE REVIEW

File: corehq/apps/locations/permissions.py [hunk 1]
```diff
@@ -43,11 +43,25 @@ def _inner(request, domain, *args, **kwargs):
     return locations_access_required(_inner)
 
 
+def get_user_location(user, domain):
+    if user.is_commcare_user():
+        return user.location
+    else:
+        return user.get_location(sql_location.domain)
```
Comment (dimagimon) [inline corehq/apps/locations/permissions.py]: F821 undefined name 'sql_location'

File: corehq/apps/locations/permissions.py [hunk 2]
```diff
@@ -43,11 +43,25 @@ def _inner(request, domain, *args, **kwargs):
...
+
+
+def get_user_sql_location(user, domain):
+    if user.is_commcare_user():
+        return user.sql_location
+    else:
+        return user.get_sql_ocation(sql_location.domain)
```
Comment (dimagimon) [inline corehq/apps/locations/permissions.py]: F821 undefined name 'sql_location'

Comment (snopoke) [inline corehq/apps/locations/permissions.py]: Also typo `get_sql_ocation`

Comment (gcapalbo) [inline corehq/apps/locations/permissions.py]: thanks, i had tested with a mobile worker and an admin web user so didn't catch this. i now retested with a read-only web user after updating

**Pull request merged**